torchsynth/module.py

```diff
@@ -354,7 +354,12 @@ def __init__(
         # TODO: Make sure this is on GPU
         # https://pytorch-lightning.readthedocs.io/en/latest/advanced/multi_gpu.html#init-tensors-using-type-as-and-register-buffer
         # Do we want to detach clone?
-        self.phase = self.get_parameter("initial_phase").detach().clone()
+        # Do we want this persistent?
+        # https://pytorch.org/docs/stable/generated/torch.nn.Module.html#torch.nn.Module.register_buffer
```

Comment:
Commented on this in other PR, but I don't think we need it persistent. If it is then it basically allows us to save and load a synth a keep these phase values right?

Reply:
Okay make a PR to fix?

Reply:
Will do